lib/collective/collectors/postgres.rb

```diff
@@ -0,0 +1,39 @@
+module Collective::Collectors
+  class Postgres < Collective::Collector
```

Comment:
The default resolution is [5s](https://github.com/remind101/collective/blob/master/lib/collective/builder.rb#L12). We don't need it to be that frequent. You can probably do once every 10 mins. 

Which also means you need to close the database connection so we don't hold on to it for a long time.